src/components/TestDateCard/index.test.tsx

```diff
@@ -5,19 +5,23 @@ import { mount, shallow } from 'enzyme';
 import { DateTime } from 'luxon';
 
 import TestDateCard from 'components/TestDateCard';
+import { TestDateTestType } from 'types/graphql-global-types';
 
 const renderComponent = (score: number | null) => {
   return mount(
     <MemoryRouter>
       <MockedProvider>
         <TestDateCard
-          date={DateTime.local().toISO()}
-          type="INITIAL"
+          testDate={{
+            __typename: 'TestDate',
```

Comment:
This feels like data logic leaking into view logic for me. I want to sit on this for a bit, I actually like the pattern we have now, in terms of how easy it is to pass props to test a thing. Let me ponder this change for a bit...

Reply:
Can you tell me more about "data logic leaking"?  I'm not sure what that means.

Reply:
I think it's okay to go with Chris's plan to pass the object. As mentioned, we have the type rigidly defined, so there's no ambiguity